src/main/java/com/icthh/xm/ms/entity/config/RestTemplateConfiguration.java

```diff
@@ -45,6 +45,11 @@ public RestTemplate loadBalancedRestTemplateWithTimeout(RestTemplateCustomizer c
         return restTemplate;
     }
 
+    @Bean
+    public RestTemplate plainRestTemplate() {
+        return new RestTemplate();
```

Comment:
pls add ability to specify readTimeout, connectionTimeout and connectionRequestTimeout, as in loadBalancedRestTemplateWithTimeout